cmd/admin_auth_ldap.go

```diff
@@ -406,14 +369,7 @@ func (a *authService) updateLdapBindDn(c *cli.Context) error {
 }
 
 // addLdapSimpleAuth adds a new LDAP (simple auth) authentication source.
-func (a *authService) addLdapSimpleAuth(c *cli.Context) error {
-	if err := argsSet(c, "name", "security-protocol", "host", "port", "user-dn", "user-filter", "email-attribute"); err != nil {
```

Comment:
I think we should keep the old `argsSet` approach, instead of duplicating flags with different "Required" values.